doc/tutorials/introduction/load_save_image/load_save_image.rst

```diff
@@ -28,6 +28,8 @@ Here it is:
 
    #include <cv.h>
    #include <highgui.h>
+   #include <opencv2/imgproc/imgproc.hpp>
+   #include <stdio.h>
```

Comment:
I prefer to use `<...>` for system or other external libraries. `"..."` is used for headers from the same project. Regarding this case OpenCV is **external library** for user application, so I will use `<...>` here.